extensions/amp-form/0.1/amp-form.js

```diff
@@ -209,7 +209,7 @@ export class AmpForm {
         this.actions_.trigger(this.form_, 'submit-error', null);
         this.setState_(FormState_.SUBMIT_ERROR);
         this.renderTemplate_(error.responseJson || {});
-        rethrowAsync('Form submission failed:', error);
+        rethrowAsync(user().createError('Form submission failed:', error));
```

Comment:
Let's have this conversation here: https://github.com/ampproject/amphtml/pull/5841#discussion_r85375956